js/stayInBackground_popup_start.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2020 Signal Messenger, LLC
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/* global $, Whisper */
+
+$(document).on('keydown', e => {
+  if (e.keyCode === 27) {
```

Comment:
Adding a comment that 27 corresponds to the escape key helps readability where constants aren't available.

Reply:
Sure. I also added a comment in other windows since this was already in 4 other places